src/py/flwr/common/message.py

```diff
@@ -25,7 +25,7 @@
 from .logger import log
 from .record import RecordSet
 
-DEFAULT_TTL = 3600
+DEFAULT_TTL = 3600 * 3
```

Comment:
Btw, I think we should have a quick discussion on this default value.

```suggestion
DEFAULT_TTL = 10800  # This is 3 hours
```